pkg/quota/evaluator/core/pods.go

```diff
@@ -33,8 +34,40 @@ import (
 	"k8s.io/kubernetes/pkg/util/validation/field"
 )
 
+// listPodsByNamespaceFuncUsingInformer returns a pod listing function based on the shared informer factory.
+func listPodsByNamespaceFuncUsingInformer(f informers.SharedInformerFactory) generic.ListFuncByNamespace {
```

Comment:
i agree the shim is unfortunate, and i would like to know a way to avoid it.  i was out of ideas at the time.